### ISSUE TITLE
Is it possible to use different styles on the same page? 

### ISSUE THREAD

**Issue by letochagone** [highlightjs/highlight.js#3731]
it is not an issue, I did not find where to ask this question (on stackoverflow?)
this question has been asked before https://github.com/highlightjs/highlight.js/issues/862
 but i didn't understand the answer, i was missing a concrete explanation of the answer

In the example below, I put two **_link_** tags in the _**body**_

Of course it doesn't work, I imagine there is a method.

```
<!doctype html>
<html>
  <head>
    <link rel="stylesheet" href="highlight/styles/codepen-embed.min.css">
    <script src="highlight/highlight.min.js"></script>
    <script>hljs.highlightAll();</script>
  </head>
  
  <body>
    
    <link rel="stylesheet" href="highlight/styles/codepen-embed.min.css">
    <pre><code>
        //javascript
        let x=5;
    </code></pre>

    <link rel="stylesheet" href="highlight/styles/arta.min.css">
    <pre><code>
        //GLSL
        uniform1i ui;
        void main() {
           gl_FragColor = vec4(1);
        }
    </code></pre>
  </body>
</html>
```

**Comment by joshgoebel** [comment]
Not trivially, but with CSS of course this wold be possible just with simple application of another class.  You'd have to modify your themes accordingly to allow this sort of thing.  That might be trivial though if you are using a pre-processor such as SCSS, etc...

```css
/* theme one */
.hljs.theme_one .hljs-variable {
  color:blue
}

/* theme two */
.hljs.theme_two .hljs-variable {
  color:red;
}
```

**Comment by letochagone** [comment]
> Not trivially, but with CSS of course this would be possible just with simple application of another class on each code block. You'd have to modify your themes accordingly to allow this sort of thing. That might be trivial though if you are using a pre-processor such as SCSS, etc...
> 
> ```css
> /* theme one */
> .hljs.theme_one .hljs-variable {
>   color:blue
> }
> 
> /* theme two */
> .hljs.theme_two .hljs-variable {
>   color:red;
> }
> ```

I don't understand your answer. In the example I give, I would like to give 2 different themes in the same page. What should be changed in this example?

**Comment by joshgoebel** [comment]
```html
<pre><code class="theme_one">
</code></pre>

<pre><code class="theme_two">
</code></pre>
```

BUT you've also have to entirely rewrite the themes CSS to have every rule nested inside a theme class also:

```css
.theme_two .hljs-tag .hljs-attr {
  color: #444;
}
```

Easy, but not trivial.

**Comment by joshgoebel** [comment]
Closing as asked and answered.  If you have any further questions feel free to still respond.
